classes/Store.php

```diff
@@ -124,7 +124,7 @@ class StoreCore extends ObjectModel
     /**
      * StoreCore constructor.
      *
-     * @param null $idStore
+     * @param int|null $idStore
      * @param null $idLang
```

Comment:
Is it worth fixing as well ?
```suggestion
     * @param int|null $idLang
```

Reply:
Fixed